intro_to_r/intro_to_r.Rmd

```diff
@@ -58,14 +58,17 @@ library(oilabs)
 library(dplyr)
 library(ggplot2)
 ```
-and finally, load in some data:
+
+
```

Comment:
I don't know! This also tripped up a bunch of my student and I struggled to explain what was going on.